### DISCUSSION TITLE
How to get the formation energy per atom for WBM dataset?

### DISCUSSION THREAD

**Discussion by rydeveraumn** [janosh/matbench-discovery#170] [Q&A]
Hello,

I have downloaded the WBM dataset and I am trying to figure out how to get the formation energy per atom?

Thank you in advance.

**Comment by rydeveraumn** [comment]
I was trying to reproduce the histogram under https://matbench-discovery.materialsproject.org/data. I found a readme and followed the additional instructions and clipped outlier formation energies:

![image](https://github.com/user-attachments/assets/a68193c6-784c-4517-8171-87a5bf3a9895)

**Comment by rydeveraumn** [comment]
I think I was able to figure it out.

**Comment by janosh** [comment]
glad you figured it out. for others coming here later, the data plotted in the above histogram is coming from the `df_summary[MbdKey.e_form_wbm]` column where `MbdKey.e_form_wbm = "e_form_per_atom_wbm"`

https://github.com/janosh/matbench-discovery/blob/93d1d5c50d99ae65f00cf255d8ffa74d2163d18c/data/wbm/compile_wbm_test_set.py#L453-L479
